README.md

```diff
@@ -159,6 +160,20 @@ x.subViews(color: y)
 x.nounCapitalize()
 ```
 
+#### Empty Return Types ####
+
+When specifying return type for functions, methods or closures that return no value, favor the type alias `Void` over empty tuple `()`.
+
+**Preferred:**
+```swift
+let onTextEntry: (String) -> Void
```

Comment:
wondering if a function with a closure would be a better (at least more common) example